Frends.AMQP.Send/Frends.AMQP.Send.Tests/Frends.Amqp.Send.Tests.cs

```diff
@@ -1,67 +1,64 @@
 using System;
 using System.Threading;
 using System.Threading.Tasks;
-using Frends.Amqp.Definitions;
+using Frends.AMQP.Send.Definitions;
 using NUnit.Framework;
 
-namespace Frends.AMQP.Send.Tests
-{
-    [TestFixture]
-    public class AMQPSendTests
-    {
-        private static readonly string queue = "task-tests";
-        private static string SecureBusAddress;
+namespace Frends.AMQP.Send.Tests;
 
-        [SetUp]
-        public void SetUp()
-        {
-            var externalAmqpsConnectionString = Environment.GetEnvironmentVariable("AMQPS_CONN_STR");
+[TestFixture]
+public class AMQPSendTests
+{
+    private static readonly string queue = "task-tests";
+    private static string SecureBusAddress;
 
-            if (string.IsNullOrWhiteSpace(externalAmqpsConnectionString))
-            {
-                TestAmqpBrokerHelper.KillTestBroker();
-                TestAmqpBrokerHelper.EnsureTestBrokerIsRunning();
-                SecureBusAddress = $"amqps://guest:guest@localhost:{TestAmqpBrokerHelper.TestAmqpBrokerPorts[1]}";
-            }
-            else
-            {
-                SecureBusAddress = externalAmqpsConnectionString;
-            }
-        }
+    [SetUp]
+    public void SetUp()
+    {
+        var externalAmqpsConnectionString = Environment.GetEnvironmentVariable("AMQPS_CONN_STR");
```

Comment:
We can use the docker container in unit tests, so the connection string can be hardcoded.

Reply:
Actually there is no need for this one when test AMQP server is up and running in docker. Removed.